.github/workflows/build.yml

```diff
@@ -0,0 +1,37 @@
+name: SonarCloud
+on:
+  push:
+    branches:
+      [main, dev, sonar-init]
```

Comment:
The `sonar-init` branch will be removed after the merge, so it is highly redundant here, jus the `main` and `dev` branches would be sufficient.